bin/broadside

```diff
@@ -89,6 +89,15 @@ command :deploy do |d|
     add_shared_deploy_configs(subcmd)
   end
 
+  d.desc 'Creates an initial task definition and service, then runs a series of bootstrap commands.'
```

Comment:
it still runs with a target, it's just not part of a `deploy`.

we have discussed - and i have actually done a little work on - the fact that not everything should be subcommand of `deploy` (is `logtail` a sub command of deploying?), but `bootstrap` is currently the only command that actually stands alone.